build.go

```diff
@@ -95,18 +95,16 @@ func build() error {
 	}
 
 	// Make the dir for the target
-	if err := os.MkdirAll(target, os.ModeDir); err != nil {
```

Comment:
os.ModeDir was creating a file as a directory (which os.MkdirAll does anyway) but with 000 permissions, causing failures later in the build.